catalog/app/components/MIcon/index.js

```diff
@@ -12,25 +15,42 @@ const Adjust = styled.span`
   top: ${(props) => props.drop || 0}
 `;
 
+const spin = keyframes`
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+`;
+
+const animation = css`
+  animation: ${spin} 1s linear infinite;
```

Comment:
we can't use `components/Spinner` for this?

Reply:
i wanted to conditionally modify the appearance of the icon that was already there, just to indicate some "alternate state", not to replace it with another one altogether. i could've put `fa-spin` class on the icon if FA was used in the menu, but since we had a mui icon there, i decided to keep it and add the required modifier to the `MIcon` component.